bin/oref0-setup.sh

```diff
@@ -180,6 +184,13 @@ if [[ -z "$DIR" || -z "$serial" ]]; then
           echo "Ok, $BT_MAC it is."
        fi
     fi
+    if [[ ! -z $BT_PEB ]]; then
+       read -p "For Pancreabble enter Pebble mac id (i.e. AA:BB:CC:DD:EE:FF) hit enter to skip " -r
+       BT_PEB=$REPLY
+       echo "Ok, $BT_PEB it is."
+    else
+       echo Ok, no Pancreabble for you.
```

Comment:
I'm just referring to the else clause. If you don't want to prompt about pancreabble unless someone passes the -p flag that is fine with me.